WTWSMS/common/landed_titles/WtWSMS_tribes.txt

```diff
@@ -952,13 +952,19 @@ d_goths = {
 	# tribe = yes
 	
 	allow = {
-		always = no
+		OR = {
+			culture = gothic
+			culture = crimean_gothic
+			culture = ostrogothic
+		}
+		controls = 560
+		prestige = 500
```

Comment:
There must have been a misunderstanding. always = no does mean that it can't be formed, which is intentional design.